plugins/module_utils/aci.py

```diff
@@ -1523,11 +1523,16 @@ def parsed_url_path(self, url):
         else:
             return parse_result.path + "?" + parse_result.query
 
+    def synthesize_url(self, httpapi_url, local_path):
+        parse_url = urlparse(httpapi_url)
+        return {"protocol": parse_url.scheme, "host": parse_url.netloc, "path": local_path}
```

Comment:
Move out of class and find a better name for function

Reply:
Done!